generate/generate.go

```diff
@@ -216,48 +230,63 @@ func generateStackLocals(
 		localsBody.SetAttributeValue(name, localsAttrs[name])
 	}
 
-	logger.Debug().
-		Msg("Write file.")
 	tfcode := AddHeader(gen.Bytes())
```

Comment:
I know this line is not changed, but this `AddHeader()` looks like it's doing something completely different. Maybe rename to `PrependHeaderBytes()` ?